app/components/view_component/form/file_field_component.rb

```diff
@@ -4,6 +4,12 @@ module ViewComponent
   module Form
     class FileFieldComponent < FieldComponent
       self.tag_klass = ActionView::Helpers::Tags::FileField
+
+      def before_render
+        @options = convert_direct_upload_option_to_url(options)
+
+        super
+      end
```

Comment:
@rowofpixels @Spone Maybe we sould add `options = { include_hidden: multiple_file_field_include_hidden }.merge!(options)` too (https://github.com/rails/rails/blob/de53ba56cab69fb9707785a397a59ac4aaee9d6f/actionview/lib/action_view/helpers/form_helper.rb#L1237-L1241).

Reply:
Good call, done.